Groups.md

```diff
@@ -1,99 +1,102 @@
-# UIC Homebrew Computer Club 小组规划（总）
+# UIC Homebrew Computer Club Groups Planning Plans
 
-_2017-8-31：本规则尚未完成。_
+| Date      | Changelog                             |
+| --------- | ------------------------------------- |
+| 2017-8-31 | This plan has not yet been completed. |
+| 2017-9-01 | Translating the Document to en-US.    |
 
-UIC Homebrew 计算机俱乐部（缩写 UICHCC）（目前）是一个非社团兴趣组织。其运作方式与 TEDxUIC 形式类似，即俱乐部内部人员负责维护俱乐部运转及参与项目开发，而**一切产出内容面向社会开放**。
+UIC Homebrew Computer Club (Abbreviation: UICHCC) is an interest-driven organization. Its mode of operation is similar to TEDxUIC. a.k.a. The club's internal member responsible for maintaining the operation of the club and participate in project development, and all the creation produce within the UICHCC will be open to the Public.
```

Comment:
"a.k.a." = "also known as", used when a noun has a more common alias. For here I think "That is, ..." will be better.